public/app/core/components/sidemenu/SideMenu.tsx

```diff
@@ -24,21 +28,87 @@ export const SideMenu: FC = React.memo(() => {
   }
 
   return (
-    <nav className="sidemenu" data-testid="sidemenu" aria-label="Main menu">
-      <a href={homeUrl} className="sidemenu__logo" key="logo">
+    <nav className={`${styles.sidemenu} sidemenu`} data-testid="sidemenu" aria-label="Main menu">
```

Comment:
Question: wouldn't it be possible to use the `cx(styles.sidemenu, 'sidemenu')` here?

Reply:
sure would 😅  great suggestion, thanks!